hack/update-vendor-licenses.sh

```diff
@@ -208,12 +205,18 @@ Options:
 __EOF__
       exit 9
   fi
-  cat "${file}"
 
-  echo
-  echo "= ${file} $(md5sum < "${file}" | awk '{print $1}')"
-  echo "================================================================================"
-  echo
-done >> ${TMP_LICENSE_FILE}
+  
+    cat "${file}"
+    echo
+    echo "= ${file} $(md5sum < "${file}" | awk '{print $1}')"
+  } >> "${TMP_LICENSE_FILE}"
 
-cat ${TMP_LICENSE_FILE} > ${VENDOR_LICENSE_FILE}
+  license_file_name=$(echo "${PACKAGE}" | sed "s/\//_/g")
```

Comment:
Yeah, in revisting this I still think a dir structure that parallels the code is better - no ambiguity and I can trivially convert paths from code files to license files.  Thanks.  The leafe files can be called LICENSE

Reply:
This comment is not addressed, and the code doesn't match the PR comments